docs/src/index.md

```diff
@@ -15,6 +15,11 @@ ParameterDistributions.jl                   | Structures to hold prior and poste
 DataContainers.jl                           | Structure to hold model parameters and outputs
 Localizers.jl                               | Covariance localization kernels
 
+## Learning the amplitude and vertical shift of a sine curve
+![Ensemble of parameter estimates by iteration](assets/sinusoid_example_params.gif)![Ensemble of resulting curves by iteration](assets/sinusoid_example_curves.gif)
```

Comment:
Also, do you think it's ok that the truth draws new samples in every frame of the gif? We don't do this in practice, but then again, it is a random function so maybe it illustrates this well. I'll let you pick!